backend/src/modules/player-profiles/presentation/rest-api/PlayerProfileRouter.ts

```diff
@@ -0,0 +1,30 @@
+import express, { Request, Response } from 'express';
+import {QueryPublisher} from "../../../../shared/core/application/query/QueryBus";
+import {FindAllPlayerProfiles, FindAllPlayerProfilesResult} from "../../core/application/query/FindAllPlayerProfiles";
+import {StatusCodes} from "http-status-codes";
+import {PlayerProfilesListDto} from "./response/PlayerProfilesListDto";
+import {PlayerProfileDto} from "./response/PlayerProfileDto";
+import {PlayerProfile} from "../../core/domain/PlayerProfile";
+
+export function playerProfileRouter(
+    queryPublisher: QueryPublisher
+):  express.Router {
+    const getAllPlayersProfiles = async (request: Request, response: Response) => {
```

Comment:
No i supcio! :)

Reply:
no co prawda jednak zacznę od POST method, ale to już tu sobie zostanie - w sumie to zapewne się i tak przyda do sprawdzenia, czy dany użytkownik już istnieje w bazie